ctx.go

```diff
@@ -63,7 +63,7 @@ type DefaultCtx struct {
 	pathOriginal        string               // Original HTTP path
 	pathBuffer          []byte               // HTTP path buffer
 	detectionPathBuffer []byte               // HTTP detectionPath buffer
-	redirectionMessages []string             // Messages of the previous redirect
+	flashMessages       redirectionMsgs      // Flash messages
```

Comment:
I don't follow why the `flashMessages` are `redirectionMsgs` when there is a flashMsg type?

Reply:
I wanted to avoid initializing 2 new slices in order not to cause new stack allocations. But i'll give a try to see how it affects the performance